src/main/java/gravitas/command/ListTaskCommand.java

```diff
@@ -0,0 +1,24 @@
+package gravitas.command;
+
+import gravitas.exception.DukeException;
+import gravitas.storage.Storage;
+import gravitas.tasklist.TaskList;
+
+/**
+ * Represents a command to list all tasks in the task list.
+ */
+public class ListTaskCommand extends Command {
+    /**
+     * Constructor for ListTaskCommand.
+     */
+    public ListTaskCommand() {
+    }
+
+    /**
+     * Executes the command to list all tasks in the task list.
+     */
+    @Override
+    public String execute(TaskList taskList, Storage storage) throws DukeException {
```

Comment:
Maybe you can change the name of the method so that it is more specific such as 'executeCommand' but overall your code is really well written!